src/main/java/com/purbon/kafka/topology/roles/RBACProvider.java

```diff
@@ -200,4 +226,31 @@ public String toString() {
     LOGGER.info("Not implemented yet!");
     return new HashMap<>();
   }
+
+  @Override
+  public List<TopologyAclBinding> setSchemaAuthorization(String principal, List<String> subjects) {
+    return subjects.stream()
+        .map(
+            subject -> {
+              try {
+                return apiClient.bind(principal, RESOURCE_OWNER).forSchemaSubject(subject).apply();
+              } catch (ConfigurationException e) {
+                LOGGER.error(e);
+              }
+              return null;
+            })
+        .filter(binding -> binding != null)
+        .collect(Collectors.toList());
+  }
```

Comment:
Hi @purbon,
just curious why is it hard-coded to `RESOURCE_OWNER`  and Pattern Type `LITERAL` for subjects rbac?
Is there some bigger problem if we allow configurable roles and pattern types for subjects in the Topology YAML?

Or is it just a matter of additional development effort, but no issues in general ?
Thanks.

Reply:
Hi @purbon I created issue #283 for the question above. 
Leonid